src/components/AddressInput.vue

```diff
@@ -89,6 +92,11 @@ export default {
       this.showList = false;
       this.$store.dispatch("SET_ADDRESS", location);
     }
+  },
+  computed: {
+    checkAddress() {
+      return this.existing ? this.existing : "Kongens slott";
```

Comment:
Could also just be `this.existing || "Kongens slott"` if you want to shorten it down a bit.

```suggestion
      return this.existing || "Kongens slott";
```

Reply:
fixed 👍🏼